deps/prereq.sh

```diff
@@ -1,21 +1,22 @@
 #/bin/sh
+
 command_exists () {
     type "$1" &> /dev/null ;
 }
 
 cd $HOME
 
 if command_exists go ; then
-    echo "Golang is already installed"
+  echo "Golang is already installed"
 else
   echo "Install dependencies"
   sudo apt update
   sudo apt-get -y upgrade
   sudo apt install build-essential jq -y
-  wget https://dl.google.com/go/go1.17.3.linux-amd64.tar.gz
-  tar -xvf go1.17.3.linux-amd64.tar.gz
+  wget https://dl.google.com/go/go1.18.1.linux-amd64.tar.gz
```

Comment:
Let's use env setting for golang version to avoid updating the script everytime.